### PR TITLE
Add quickstart guide for workdocs

### PR DESCRIPTION
**Pull request by yuhgto**
Added a quickstart guide for workdocs. 

Please check the following files: 
- apps/quickstart-workdocs/src/App.js - this contains the app's main logic
- apps/quickstart-workdocs/package.json - changed the start scripts, port, etc
- apps/quickstart-workdocs/README.md - minimalist readme file

The rest of the project is boilerplate from create-react-app, or copied from the quickstart-react project.

### CODE REVIEW

File: apps/quickstart-workdocs/package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "monday-workdocs-quickstart-app",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "start": "npm run stop && concurrently \"npm run server\" \"npm run expose\"",
+    "build": "react-scripts build",
+    "expose": "mtunnel -p 8306",
```
Comment (gregra81) [inline apps/quickstart-workdocs/package.json:8]: there are repeating issues with our tunnel solution. I suggest removing it altogether from our samples and documentation and suggest ngrok. @arutkowski00 FYI

File: apps/quickstart-workdocs/package.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    "server": "react-scripts start",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject",
+    "stop": "kill-port 8301 && kill-port 4049 && kill-port 4040"
+  },
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.5.0",
+    "eslint-config-react-app": "^7.0.1",
```
Comment (gregra81) [inline apps/quickstart-workdocs/package.json:17]: these should go to `devDependencies`

File: apps/quickstart-workdocs/package.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    "monday-sdk-js": "^0.1.4",
+    "monday-ui-react-core": "^1.54.0",
+    "quill-delta": "^5.0.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-scripts": "^5.0.1"
```
Comment (gregra81) [inline apps/quickstart-workdocs/package.json:23]: this should go to `devDependencies`

File: apps/quickstart-workdocs/public/index.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (gregra81) [inline apps/quickstart-workdocs/public/index.html:27]: Let's change the name to "Monday Docs Example"

**Review by gregra81 — CHANGES_REQUESTED** [review]
Looks great! 🥇 
Left you some comments

File: apps/quickstart-workdocs/src/App.js [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+            const ret = {
+              insert: newInsert,
+              attributes: op?.attributes
+            }
+            return ret;
+          }
+          else if (typeof op.insert === 'string') {
+            const ret = {
+              insert: op.insert.toUpperCase(),
+              attributes: op?.attributes
```
[the remark this answers is not in the document]
Comment (yuhgto) [reply apps/quickstart-workdocs/src/App.js:42]: Not in this case: it's from the delta-format. All delta formatted content has an optional attributes key. 

From the delta docs: 
image

File: apps/quickstart-workdocs/package.json (hunk 1 above)
Comment (yuhgto) [reply apps/quickstart-workdocs/package.json:8]: Good call. Will remove the script for now but keep it in the dependencies as an alternative to ngrok.

Comment (yuhgto) [reply apps/quickstart-workdocs/package.json:8]: I also added an "Install ngrok" step to the article: 
image

File: apps/quickstart-workdocs/package.json (hunk 2 above)
Comment (yuhgto) [reply apps/quickstart-workdocs/package.json:17]: Done!

File: apps/quickstart-workdocs/package.json (hunk 3 above)
Comment (yuhgto) [reply apps/quickstart-workdocs/package.json:24]: Done!

File: apps/quickstart-workdocs/public/index.html (hunk 4 above)
Comment (yuhgto) [reply apps/quickstart-workdocs/public/index.html]: Done!

**Comment by yuhgto** [comment]
@gregra81 – changed made, ready for your review

**Review by rami-monday — APPROVED** [review]

**Review by vladmonday — APPROVED** [review]

**Pull request merged**